README.adoc

```diff
@@ -0,0 +1,1763 @@
+= The RSpec Style Guide
+:doctype: article
+
+This RSpec style guide outlines the recommended best practices for real-world
+programmers to write code that can be maintained by other real-world
+programmers.
+
+https://github.com/rubocop-hq/rubocop[RuboCop], a static code analyzer
+(linter) and formatter, has a
+https://github.com/rubocop-hq/rubocop-rspec[`rubocop-rspec`] extension,
+provides a way to enforce the rules outlined in this guide.
+
+This guide assumes you are using RSpec 3 or later.
+
+== How to Read This Guide
+
+The guide is separated into sections based on the different pieces of an entire
+spec file. There was an attempt to omit all obvious information, if anything is
+unclear, feel free to open an issue asking for further clarity.
+
+== A Living Document
+
+Per the comment above, this guide is a work in progress - some rules are simply
+lacking thorough examples, but some things in the RSpec world change week by
+week or month by month. With that said, as the standard changes this guide is
+meant to be able to change with it.
+
+== Table of Contents
```

Comment:
You don't need to hardcode the table of contents. This feature is already provided by AsciiDoc. Add `:toc: macro` to the document header, then use `toc::[]` to place the table of contents.

Reply:
Again, set the `:toc: macro` in the document header, then put `toc::[]` whereever you want the ToC to be placed.

In other words:

```
= Document Title
:toc: macro

yada

yada

toc::[]

yada

yada
```